lib/resolveurl/__init__.py

```diff
@@ -310,7 +310,7 @@ def _update_settings_xml():
         old_xml = u''
 
```

Comment:
I'd suggest
```suggestion
    old_xml = six.ensure_text(old_xml)
```
instead.

Reply:
Yes we already use `six.ensure_text` for new.xml. Would make sense to use it for old.xml as well
Thanks